content/applications/sales/sales/send_quotations/orders_and_variants.rst

```diff
@@ -1,79 +1,121 @@
-===================================================
-Use product variants on quotations and sales orders
-===================================================
+===============================================
+Product variants on quotations and sales orders
+===============================================
 
-Product variants can be added to quotations and sales orders using the Product Configurator,
-Variant Grid Entry, or both.
+Before getting into detail about how to use product variants on quotations and sales orders, it's
+recommended to learn about :doc:`/applications/sales/sales/products_prices/products/variants` in
+Odoo.
 
-Configuration
-=============
+Once familiarized with the basics surrounding product variants, the following covers how product
+variants can be added to quotations and sales orders using the *Product Configurator* or *Order Grid
+Entry*.
 
-To enable product variants on quotations and sales orders, go to :menuselection:`Sales -->
-Configuration --> Settings --> Product Catalog` and activate one or both of the
-:guilabel:`Product Configurator` and :guilabel:`Variant Grid Entry` settings. Finally, click
-:guilabel:`Save` to apply the settings.
+.. note::
+   It should be noted that the setting is titled, `Variant Grid Entry` on the *Sales* app settings
+   page, and titled, `Order Grid Entry` on product forms. So, be sure to keep that in mind.
```

Comment:
Just wondering why the "Order Grid Entry" and "Variant Grid Entry" features are surrounded with the back-tick (`) here? Is it to make these feature names stand out in the note admonition block?

Was just wondering about your rationale for this formatting choice, since we use guilabels when referring to UI items (not what you're doing here), [italics for concepts](https://docs.google.com/document/d/1laiWC9ow0IjXPuQp5h8L_Tj98HbE8ymGn9v-FFOwlYM/edit#heading=h.90nibixh8llr), [back-ticks for user entries](https://docs.google.com/document/d/1laiWC9ow0IjXPuQp5h8L_Tj98HbE8ymGn9v-FFOwlYM/edit#heading=h.ipar7km38ci3)

Just wondering how consistent should we be in our formatting choices? Also when you get to this comment, can you let me know your thoughts in a Discord DM? I want to know what you think, and I miss these github conversations a lot

Reply:
Great question. So, my initial intent was to use ticks because these elements aren't directly on the screen that's being explained. however, I do believe you're correct -- these should be GUI, so I'll make that change right away. Thanks for pointing that out.